software/Makefile.kernel

```diff
@@ -29,7 +29,7 @@ $(KERNEL_IMAGE): kernel/.git/HEAD
 kernel: $(KERNEL_IMAGE)
 clean-kernel:
 	-$(MAKE) -C kernel clean
-	-rm -f $(KERNEL_IMAGE)
+	$(RM) $(KERNEL_IMAGE)
```

Comment:
Out of curiosity, why do you prefer `$(RM)` to `-rm -f`?

Reply:
I don't, I'm just assuming make knows better (maybe for supporting specific platforms?)